puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/vars/katello/3.16.groovy

```diff
@@ -3,3 +3,10 @@ def katello_version = '3.16'
 def foreman_el_releases = [
     'el7'
 ]
+def katello_server_distros = [
+    'centos7',
+    'centos8'
```

Comment:
This is odd since `foreman_el_releases` doesn't include el8 which means the RPMs are not being published. Is this intentional?